PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m

```diff
@@ -49,9 +49,8 @@ - (PNLiteAdRequestModel *)createAdRequestWithZoneID:(NSString *)zoneID
     [self setIDFA:adRequestModel];
     adRequestModel.requestParameters[HyBidRequestParameter.locale] = [HyBidSettings sharedInstance].locale;
     
-//    Commenting out the following 3 parameters for now since the backend is not supporting those yet.
-//    adRequestModel.requestParameters[HyBidRequestParameter.versionOfOMSDKIntegration] = HYBID_OMSDK_VERSION;
-//    adRequestModel.requestParameters[HyBidRequestParameter.identifierOfOMSDKIntegration] = HYBID_OMSDK_IDENTIFIER;
+    adRequestModel.requestParameters[HyBidRequestParameter.versionOfOMSDKIntegration] = HYBID_OMSDK_VERSION;
+    adRequestModel.requestParameters[HyBidRequestParameter.identifierOfOMSDKIntegration] = HYBID_OMSDK_IDENTIFIER;
```

Comment:
Are those fields supported in our API? Is it tested? During my implementation it was not supported and ad requests were failing.

Reply:
Now they are. It was released on the ad server a few days ago.